tests/hilti/output/optimization/const.hlt

```diff
@@ -0,0 +1,113 @@
+# @TEST-DOC: Tests optimizations performing constant folding.
+#
+# @TEST-REQUIRES: which FileCheck
+# @TEST-EXEC: hiltic %INPUT -pg | FileCheck %INPUT -check-prefix=NOOPT
+# @TEST-EXEC: hiltic %INPUT -p | FileCheck %INPUT -check-prefix=OPT
+# @TEST-EXEC: hiltic %INPUT -p -D optimizer
+# @TEST-EXEC: cat .stderr | sort -k 3 > log
+# @TEST-EXEC: btest-diff log
```

Comment:
For many of the optimizer tests, the debug baselines are very sensitive to changes, and need updating all the time. So maybe when switching to FileCheck, we could also stop diffing the debug output?